README.md

```diff
@@ -18,6 +23,13 @@ Open a notepad in the Firefox toolbar to store notes about your browsing.
 | `npm run build` | Builds the application as a Web Extension.
 | `npm start`     | Launches Firefox with the Web Extension.
 
+NOTE: if you are running on Windows, and experience a build error use `$ npm build`
```

Comment:
Can you try again `npm run build` with this new update? Here (on linux) `npm build` doesn't do anything.

Also if it fails with `npm run build` on windows it should fail with `npm build` as well because the error is not related to npm but to windows.